### ISSUE TITLE
Default theme has Transparent Panel, is it a feature or bug?

### ISSUE THREAD

**Issue by fastrizwaan** [solus-project/budgie-desktop#291]
![screenshot from 2016-01-02 08-56-52](https://cloud.githubusercontent.com/assets/1161682/12073113/0de746ba-b130-11e5-9aeb-9bfed9377416.png)

with the Default theme the applications menu has good themeing, but the panel is 100% transparent.

with the Built-in theme the app menu looks just grey everywhere, but panel is opaque.

Can we have non-transparent panel with the default theme?

Please observe the differences between  Built-in (all flat and dark) and default (colorful, visually appealing)
![menus](https://cloud.githubusercontent.com/assets/1161682/12073158/0bc12a6a-b133-11e5-9415-5776044eaff3.png)

**Comment by ikeydoherty** [comment]
OK so the menu actually looks shite with the theming disabled, that's why we override it.
And the theme provides the panel theme...

This option is provided so *theme authors* can provide alternative themes for Budgie, and they can override all the default theming, instead of us.

We have to do this because *no themes* exist for Budgie. The GTK theme doesn't have what we need.

TL;DR: You turned off the theme and it looks bad. That's the expected behaviour.
